### ISSUE TITLE
Add an additional customisable shortcut for external apps

### ISSUE THREAD

**Issue by LukasMFR** [UpsilonNumworks/Upsilon#228]
**Is your feature request related to a problem? Please describe.**
It's a really a problem but see :

**Describe the solution you'd like**
KhiCAS is is a math-oriented app of course so people may want to quickly access it.
In KHI HOME selects the Calculation app, and BACK selects KhiCAS.
Adding the same behavior in Upsilon would be great.

The best way would be to see if an app exists (not sure if this is possible) and select, but selecting the coordinates is of course possible, I think that KhiCAS is usually the first external app (unless you have BadApple as it has a B but not sure how it organizes them). So generally KhiCAS has a fixed position, but adding the possibility of manually selecting the coordinates would be a great idea (because for instance, I choose to remove RPN, Atomic, and Reader as I don't use them, so the coordinates are different but I can edit this in code.

Also maybe adding Shift+Something for external apps may be a good idea but I tried to do that in the source code and it crashes the calculator. And also apps already use Shift+Something like KhiCAS.

**Comment by LukasMFR** [comment]
Here is somehow how I imagined it (partially)
In French but you get the idea.

![Capture d’écran 2022-05-09 à 22 01 20](https://user-images.githubusercontent.com/41242166/167488361-b7c7e683-1e09-4c9d-af20-b7414585adde.png)

**Comment by Yaya-Cout** [comment]
I think it can be a setting in the external submenu

**Comment by Lauryy06** [comment]
Yes, it is strange to favor a specific external app over the others, but eventually a settings menu can be used to define an additional shortcut.
